ravenpy/models/emulators/sacsma.py

```diff
@@ -0,0 +1,672 @@
+        #                                            soil saturation conditions.; percent/100; default=0.0
+        # para_x15 0.0    0.1      # RIVA=SARVA;     A decimal fraction representing that portion of the basin normally covered by streams,
+        #                                            lakes and vegetation that can deplete stream flow by evapotranspiration.; percent/100;
+        #                                            default=0.0
+        # para_x16 0.0    0.4      # RSERV;          Fraction of lower zone free water unavailable for transpiration; percent/100; default=0.3
+        # n/a      0.0    1.0      # UH1;            The first  component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # n/a      0.0    1.0      # UH2;            The second component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # n/a      0.0    1.0      # UH3;            The third  component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # n/a      0.0    1.0      # UH4;            The fourth component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # n/a      0.0    1.0      # UH5;            The fifth  component of the unit hydrograph, i.e. the proportion of instantaneous runoff
+        #                                            not lagged; percent/100; default=1.0
+        # para_x17 0.0    8.0      # MELT_FACTOR;    maximum snow melt factor used in degree day models (not in original SAC-SMA
+        #                                            model); mm/d/C; default=1.5
+        # para_x18 0.3    20.0     # GAMMA_SHAPE;    used to build unit hydrograph, LandUseParameterList; none ; default=1.0
+        # para_x19 0.01   5.0      # GAMMA_SCALE;    used to build unit hydrograph, LandUseParameterList; none ; default=1.0
+        # para_x20 0.8    1.2      # RAINCORRECTION; Muliplier to correct rain, Gauge; none ; default=1.0
+        # para_x21 0.8    1.2      # SNOWCORRECTION; Muliplier to correct snow, Gauge; none ; default=1.0
+
+        # tied parameters:
+        # (it is important for OSTRICH to find every parameter place holder somewhere in this file)
+        # (without this parameters that are used to derive parameters wouldn't be detectable by Ostrich)
+        #    para_x04_mm       = par_soil0_mm     = para_x04 * 1000       =  par_x04 * 1000
+        #    para_x06_mm       = par_soil0_mm     = para_x06 * 1000       =  par_x06 * 1000
```

Comment:
should be:
```
para_x06_mm = {derived_params.par_soil2_mm} = para_x06 * 1000 = {params.par_x06} * 1000
```
such that values get filled in (see also typo for `par_soil0_mm` instead of `par_soil2_mm`)